analyses/rna-seq-expression-summary-stats/01-tpm-summary-stats.R

```diff
@@ -376,24 +376,44 @@ stopifnot(identical(
 ))
 
 
-# independent sample table
-rna_idps_df <- read_tsv(
+# all cohorts independent sample table
+rna_ac_idps_df <- read_tsv(
+  file.path('..', 'independent-samples', 'results',
+            'independent-specimens.rnaseq.primary.tsv'),
+  col_types = cols(.default = col_guess()))
+
+rna_ac_idps_kfbids <- rna_ac_idps_df$Kids_First_Biospecimen_ID
+# assert no NA in rna_ac_idps_kfbids
+stopifnot(identical(as.integer(0), sum(is.na(rna_ac_idps_kfbids))))
+# assert all rna_ac_idps_kfbids are unique
+stopifnot(identical(length(rna_ac_idps_kfbids),
+  length(unique(rna_ac_idps_kfbids))))
+
+# subset histology data frame to have all cohorts independent samples in tpm_df
+rna_ac_htl_df <- htl_df %>%
+  filter(Kids_First_Biospecimen_ID %in% colnames(tpm_df),
+         Kids_First_Biospecimen_ID %in% rna_ac_idps_kfbids)
+stopifnot(all(rna_ac_htl_df$Kids_First_Biospecimen_ID %in% colnames(tpm_df)))
+
+# each cohort independent sample table
+rna_ec_idps_df <- read_tsv(
   file.path('..', 'independent-samples', 'results',
```

Comment:
```suggestion
  file.path('../..', 'data',
```
We can change the file path now that we release independent samples